tests/defences/detector/poison/test_ground_truth_evaluator.py

```diff
@@ -20,203 +20,213 @@
 import json
 import logging
 import pprint
-import unittest
 
-from art.defences.detector.poison import GroundTruthEvaluator
+import pytest
 
-from tests.utils import master_seed
+from art.defences.detector.poison import GroundTruthEvaluator
 
 logger = logging.getLogger(__name__)
 
+n_classes = 3
```

Comment:
I think we should decorate all agnostic tests with `@pytest.mark.framework_agnostic` to make sure they run only once.
We also need to wrap tests into
```
    try:
       ...
    except ARTTestException as e:
        art_warning(e)
```

Reply:
Fixed in commit